cli.js

```diff
@@ -141,90 +139,132 @@ function startProgressLogging (stats) {
   })
 }
 
-function printScanProgress (stats, last) {
-  var dirCount = stats.directories + 1 // parent folder
+function printScanProgress (stats, opts) {
+  if (!opts) opts = {}
+  var dirCount = stats.total.directories + 1 // parent folder
   var msg = chalk.bold.blue('Calculating Size: ')
-  if (last) msg = chalk.bold.green('Creating Dat Link ')
-  msg += chalk.black(
```

Comment:
@maxogden I changed this to `bold`. Unfortunately, black/white don't really work on chalk because they don't change for terminal background colors. So I couldn't see black at all on a dark terminal =(. If bold is too much, we can use the default output color.